rust_src/src/editfns.rs

```diff
@@ -46,6 +46,13 @@ pub fn point() -> EmacsInt {
     buffer_ref.pt as EmacsInt
 }
 
```

Comment:
These belong in buffers.rs with the others.

Reply:
With `BEG` and `BEG_BYTE`, you mean? Can do.